knip.ts

```diff
@@ -89,26 +84,14 @@ export default {
       project: [...defaultWorkspaceProjectConfig.project, "!config/*"],
     },
     "packages/http/tools/http-benchmarks": {
-      entry: defaultWorkspaceProjectConfig.entry,
-      ignoreDependencies: [
-        "@inversifyjs/http-core",
-        "@inversifyjs/http-express",
-        "@nestjs/common",
-        "@nestjs/core",
-        "@nestjs/platform-express",
-        "@nestjs/platform-fastify",
-        "@types/express",
-        "@types/k6",
-        "express",
-        "fastify",
-        "inversify",
-        "rxjs"
-      ],
-      ignore: [
-        "src/constant/*.ts",
-        "src/scenario/*/setUp*.ts",
+      entry: [
+        "src/bin/run.ts",
         "src/k6/scenario/*.ts",
-        "src/scenario/nestJS/**/*.ts",
+        "src/scenario/*/setup.ts",
+      ],
```

Comment:
@adrianmjim this way we tell knip which entrypoints should be analyzed. If we don't provide them, knip is likelly to fail detecting unused content